.sonarlint/sonarscanner-msbuildCSharp.ruleset

```diff
@@ -1,379 +1,365 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I compared it with the one on the analyzer side and they do not match is the different configuration intentional?

Reply:
maximumNestingLevel - sonar-dotnet is wrong about this, rule S134 reads maximumNestingLevel
License header should be different, as it started different year and has different project name.

Reply:
Fix on `sonar-dotnet` side is on its way: https://github.com/SonarSource/sonar-dotnet/pull/4813